bbq/compiler/compiler_test.go

```diff
@@ -33,6 +33,8 @@ func TestCompileRecursionFib(t *testing.T) {
 
 	t.Parallel()
 
+	t.SkipNow()
```

Comment:
Skipping these for now, since the generated code's structure is still changing (e.g: code generation for returns, etc.), and having to change these tests each and every time is not worth the hassle. We can enable these at the end, once the code generation become somewhat finalized.

Reply:
I guess that's fine, as long as we keep the "integration tests" in the VM, i.e. testing executing the compiled code still works.

Reply:
Created an issue https://github.com/onflow/cadence/issues/3758 so we don't forget.